src/components/Footer.tsx

```diff
@@ -72,31 +75,44 @@ const Footer: React.FC = () => {
               <a
                 href="https://github.com/Code-4-Community"
                 className={classes.footerLink}
+                target="_blank"
+                rel="noopener noreferrer"
               >
                 Github
               </a>
               {' | '}
-              <a href="https://c4cneu.slack.com" className={classes.footerLink}>
+              <a
+                href="https://c4cneu.slack.com"
+                className={classes.footerLink}
+                target="_blank"
+                rel="noopener noreferrer"
```

Comment:
what's this do?

Reply:
target blank is open in a new tab, the rel stuff is needed to prevent some kind of XSS attack that eslint will warn you about